CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
```

Comment:
Notably, this project did not have a CHANGELOG for all of version 0 nor 1. Perhaps this should say, "starting with v2.0.0, all notable..."